mxd/docs/File Transfer Azure to Azure.md

```diff
@@ -0,0 +1,416 @@
+# File Transfer: Azure Storage to Azure Storage
+
+## 1. Description
+This tutorial illustrates the process through which a provider participant can transfer a file stored in its Azure Storage to another Azure Storage belonging to a consumer participant.  
```

Comment:
the official name is "Azure Blob Storage", so we should use that in documentation. ref: https://azure.microsoft.com/en-us/products/storage/blobs

Reply:
Referred `Azure Blob Storage` in the documentation.